src/iop/useless.c

```diff
@@ -21,18 +21,37 @@
 // our includes go first:
 #include "bauhaus/bauhaus.h"
 #include "develop/imageop.h"
+#include "develop/imageop_gui.h"
 #include "gui/gtk.h"
 #include "iop/iop_api.h"
 
 #include <gtk/gtk.h>
 #include <stdlib.h>
 
+// This is an example implementation of an image operation module that does nothing usefull.
+// It demonstrates how the different functions work together. To build your own module,
+// take only the required functions, stripping them of comments, and add the ones to implement
+// the functionality you need. Don't copy default implementations (hint: if you don't need to change
```

Comment:
```suggestion
// take only the required functions, stripping them of comments, and add new functions to implement
// any additional functionality. Don't copy default implementations (hint: if you don't need to change
```

Reply:
I think that's not exactly what I meant to say. Maybe this:
"to build your own module, take all of the functions that are mandatory, stripping them of comments. then add only the the optional functions that are required to implement the functionality you need. don't copy default implementations (hint: if you don't need to change anything, you probably don't need the copy)

Reply:
That sounds fine. Add some capital letters at the start of sentences though please!

Reply:
I'm not sure I'm allowed to! @TurboGit what is the rule around capitals in _d_arktable source precisely? or is there no rule?

Reply:
Capital letters in comment blocks should be fine. There are some already - just not consistently. I'm not suggesting adding any to the actual code but you're basically giving big chunks of prose here so proper sentences (_d_arktable notwitstanding) shouldn't be an issue.

All lower case is fine if it's a one-liner.